builder/main.py

```
@@ -56,6 +56,189 @@
+        try:
+            # Find pipenv executable first
+            pipenv_executable = None
+            
+            # Get Python version for macOS user paths
+            python_version = None
+            try:
+                result = subprocess.run([python_exe, "-c", "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')"], 
+                                      capture_output=True, text=True)
+                if result.returncode == 0:
+                    python_version = result.stdout.strip()
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)